index.js

```diff
@@ -1,53 +1,42 @@
 const dogs = []
-document.addEventListener('DOMContentLoaded', () => {
-    fetch('http://localhost:3000/dogs') 
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
You had single quotes here, but double quotes everywhere else.

This is minor; I wouldn't block code from being merged at work for this, but consistency can help a lot when you do a search for something.